test/declaration_test.ts

```diff
@@ -133,6 +133,17 @@ describe('interfaces', () => {
   });
 });
 
+describe('function types', () => {
```

Comment:
The tests usually describe the TypeScript construct that's supported, so this should be `describe('single call signature interfaces', () ...)`.